boards/arm/mec1501modular_assy6885/CMakeLists.txt

```diff
@@ -6,3 +6,48 @@
 
 zephyr_library()
 zephyr_library_sources(pinmux.c)
+
+if(DEFINED ENV{EVERGLADES_SPI_GEN})
+  # Grab it from environment variable if defined
+  set(EVERGLADES_SPI_GEN $ENV{EVERGLADES_SPI_GEN})
```

Comment:
This needs documentation.

Reply:
@SebastianBoe would it be enough to mention in the documentation that we should export EVERGLADES_SPI_GEN?

Reply:
yes.

Reply:
Added more info in comment here and reference to board documentation for futher details.
SPI generation tool documentation is added as part board doc update PR https://github.com/zephyrproject-rtos/zephyr/pull/19731/